ui/selectmenu.js

```diff
@@ -104,7 +104,7 @@ return $.widget( "ui.selectmenu", {
 			.insertAfter( this.element );
 
 		this._addClass( this.button, "ui-selectmenu-button ui-selectmenu-button-closed",
-			"ui-widget ui-state-default" );
+			"ui-button ui-icon-end ui-widget ui-state-default" );
```

Comment:
Is `ui-state-default` still needed here, since `ui-button` now gets the same styles as `ui-state-default`? Which make me wonder, why did we replace it in button?